CMakeLists.txt

```diff
@@ -353,91 +356,112 @@ if (OPENAL)
   endif()
 endif()
 
+set(BASE_DATA_DIR "./" CACHE INTERNAL "Base data dir")
```

Comment:
Would be nice to have this as an option of some sort too. Not sure there's a nice way to provide an interface for this using cmake though…

Reply:
This is the best I found so far not ideal maybe but that avoids using programmatically API not everywhere available to get the assets.